dbt/include/firebolt/macros/utils/cast_bool_to_text.sql

```diff
@@ -0,0 +1,7 @@
+{% macro firebolt__cast_bool_to_text(field) %}
+    CASE
+        WHEN {{ field }} = 0 THEN 'false'
+        WHEN {{ field }} = 1 THEN 'true'
```

Comment:
Should we consider any non-zero value as true?

Reply:
I believe the idea here is that underlying column is already boolean. It should only hold true/false (or as we store them 0 or 1). Most other adapters rely on CAST({{ field }} AS STRING) here, which would work correctly only with 'true/false' - https://github.com/dbt-labs/dbt-core/blob/main/core/dbt/include/global_project/macros/utils/cast_bool_to_text.sql#L6 . Redshift overrides the behaviour but also accounts for only true/false https://github.com/dbt-labs/dbt-redshift/blob/main/dbt/include/redshift/macros/utils/cast_bool_to_text.sql#L2 
Therefore, unless there's a particular usecase, I'd rather not introduce unexpected functionality in here.